incubator/hnc/internal/reconcilers/object_test.go

```diff
@@ -40,6 +41,7 @@ var _ = Describe("Exceptions", func() {
 			name         string
 			selector     string
 			treeSelector string
+			noneSelector bool
```

Comment:
Better to make this a string like the others, that way you can test values other than `true` or `false` which is probably a good test to add here anyway.

Reply:
done